Tools/clinic/clinic.py

```diff
@@ -865,6 +865,14 @@ class CLanguage(Language):
         PyDoc_STRVAR({c_basename}__doc__,
         {docstring});
     """)
+    GETSET_DOCSTRING_PROTOTYPE_VAR: Final[str] = normalize_snippet("""
+        PyDoc_VAR({getset_basename}__doc__);
+    """)
```

Comment:
I don't think this is needed; it does not end up being used at all in the generated code. Moreover, this macro is hardly used directly in the code base. It's use is implicit via `PyDoc_STRVAR`.

Reply:
BTW, this also goes for the already existing `DOCSTRING_PROTOTYPE_VAR`; it is never used.

Reply:
Yeah agree :) Thanks for catching up!

Reply:
For removing `DOCSTRING_PROTOTYPE_VAR `it needs modifying some test codes, I will let this variable as a separated issue(Since you might want to clean up in your way :))

Reply:
Yeah, don't worry about `DOCSTRING_PROTOTYPE_VAR`. We can look into it later. Also, let's keep Chesterton's Fence in mind before tearing it out.